README.md

```diff
@@ -663,6 +663,7 @@ Inspired by [awesome-php](https://github.com/ziadoz/awesome-php).
 
 * [grequests](https://github.com/kennethreitz/grequests) - requests + gevent for asynchronous HTTP requests.
 * [httplib2](https://github.com/httplib2/httplib2) - Comprehensive HTTP client library.
+* [httpx](https://github.com/encode/httpx) - A next generation HTTP client for Python.
```

Comment:
I *think* the name is supposed to be uppercased:

```suggestion
* [HTTPX](https://github.com/encode/httpx) - A next generation HTTP client for Python.
```

Reply:
Changed, amended, and force-pushed.